.github/workflows/deploy.yml

```diff
@@ -0,0 +1,68 @@
+name: Dev deployment from Github to AWS
+on:
```

Comment:
Re-commenting to link to relevant lines:
Does this run when a tagged commit is merged into master, or anytime there's a PR with a tag? - been a while since I looked into gha triggers